app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
+		type: Sequelize.STRING,
+		allowNull: false
+	},
+	dispatchedAt: {
+		type: Sequelize.DATE,
+		allowNull: false,
+		field: 'dispatched_at'
+	},
+};
```

Comment:
Works for me -- we just need to be 100% sure that client side cannot be manipulated without a message from the server (i.e. enabling SSL on IRC, for instance, would result in a message from the server to that client saying "yep, you enabled SSL)

Solid plan, just need to keep an eye out for it collectively to make sure we don't cheat!